components/layout/flow.rs

```diff
@@ -1115,6 +1116,30 @@ impl BaseFlow {
         }
     }
 
+    /// Update the 'flags' field when computed styles have changed.
+    ///
+    /// These flags are initially set during flow construction.  They only need to be updated here
+    /// if they are based on properties that can change triggering `RECONSTRUCT_FLOW`.
+    pub fn update_flags(&mut self, style: &ServoComputedValues) {
+        if !self.restyle_damage.contains(UPDATE_FLOW_FLAGS) {
+            return
+        }
+
+        // For absolutely-positioned flows, changes to top/bottom/left/right can cause these flags
+        // to get out of date:
```

Comment:
Perhaps note that we don't need to update the `IS_ABSOLUTELY_POSITIONED` flag because we reconstruct on position changes?